### DISCUSSION TITLE
Latest Distribution in the Web sidebar File count and size are not displayed?

### DISCUSSION THREAD

**Discussion by tagerwang** [verdaccio/verdaccio#2922] [Q&A] [answered]
os: MacOS Big Sur / Apple M1
platform: npm
try: Verdaccio tried using V5 and V6, node environments from V12 to V16, The result is the same! 

**This is shown in a screenshot of the official documentation：https://verdaccio.org/zh-CN/docs/webui: **
![image](https://user-images.githubusercontent.com/27122038/150065668-0b97bb91-292b-47b6-b368-5b5eb0d91675.png)

**Not shown on the local server: **
![image](https://user-images.githubusercontent.com/27122038/150066112-47465e24-6c19-45fc-a468-757683360d91.png)

It doesn't exist in Verdaccio Storage either! why?
![image](https://user-images.githubusercontent.com/27122038/150073478-d5ef8dca-38ff-466f-9899-0b362604a526.png)

**Comment by mbtools** [comment, marked as answer]
https://github.com/verdaccio/verdaccio/pull/4600
